container/mock_docker_client.go

```diff
@@ -0,0 +1,89 @@
+// +build ignore
+package container
+
+import (
+	"errors"
+
+	"github.com/fsouza/go-dockerclient"
+)
+
+type MockDockerClient struct {
```

Comment:
Or maybe change its package name to `container_test` and place it in a directory called `internal`?